### PR TITLE
Moment to dayjs

### PR DESCRIPTION
**Pull request by kjkurtz**
This shouldn't actually change any functionality, just standardize our time library to dayjs.

### CODE REVIEW

File: packages/ui-demo/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "@react-native-community/blur": "^4.3.0",
     "@react-native-community/datetimepicker": "6.7.3",
     "@react-native-picker/picker": "2.4.8",
+    "dayjs": "^1.11.9",
```
Comment (joshgachnang) [inline packages/ui-demo/package.json:43]: Can we remove moment as a dep?

File: packages/ui-demo/src/TextField.stories.tsx [hunk 2]
```diff
@@ -1,9 +1,12 @@
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
 import {Text, TextField} from "ferns-ui";
-import moment from "moment-timezone";
 import React, {ReactElement, useState} from "react";
 
 import {StorybookContainer} from "./StorybookContainer";
 
+dayjs.extend(utc);
```
Comment (joshgachnang) [inline packages/ui-demo/src/TextField.stories.tsx:8]: Do we still need this, or can we use the extended one below?

**Review by joshgachnang — APPROVED** [review]
Nice, that was pretty easy! Nice that they have basically the same DX.

File: packages/ui-demo/package.json (hunk 1 above)
Comment (kjkurtz) [reply packages/ui-demo/package.json:43]: Good call! Did in ui but not demo

File: packages/ui-demo/src/TextField.stories.tsx (hunk 2 above)
Comment (kjkurtz) [reply packages/ui-demo/src/TextField.stories.tsx:8]: This is just because it's within `ui-demo` and our wrapper is scoped to `ui`.

**Pull request merged**